entries/e/engadget.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Engadget": {
+    "domain": "engadget.com",
+    "url": "https://www.engadget.com/",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://login.engadget.com/myaccount/security/",
```

Comment:
The documentation is only accessible for logged-in users. Is there public documentation availlable? Alternatively, can you provide screenshots of the setup process (without any personal information)?

Reply:
I couldn't really find any public or private documentation on this feature, as it's automatically enabled when you make an account.

Reply:
What could I exactly do?

Reply:
Remove the documentation link

Reply:
Changes made.